### DISCUSSION TITLE
Cut & Paste file/folders with sandboxed Windows Explorer

### DISCUSSION THREAD

**Discussion by SandboxerX86** [sandboxie-plus/Sandboxie#3524] [General]
It may be a stupid question but I have a vague memory that in pre-1.0 versions of Sandboxie Plus when you Cut a file/folder that was located on the host with an sandboxed Windows Explorer and pasted it in an unSandboxed folder what it did was a copy of the file/folder instead of actually moving the original file from the host like it does now. Was this like that or do I remember it wrong? Isn't it better to make a copy instead of actually moving it? since the cut is made with the sandboxed explorer it should not touch the file/folder as it is on the host.

**Comment by bastik-1001** [comment]
I don't know how it used to be, as I never tried that, but I can reproduce the behavior on Windows 7. I assume that this is true for other versions of Windows

To me, it is not surprising that this happens, since the action is done by something that is not sandboxed (the explorer outside the box). I am not even sure, that Sandboxie can interfere with that. If it can, it should have the option, preferably enabled by default, to stop it.

**Reply by SandboxerX86** [reply]
Yeah, to me it makes sense too but for some reason I have this memory of it working this way and I don't know why so that's why I wanted to make sure. Maybe it was like that on legacy Sandboxie? who knows.

I don't know what has preference in this case, whether Sandboxie, which is the one that has done the Cut action with the sandboxed Explorer, or Windows that does the pasting.
In any case, I agree with you that if it is possible for it to work like this, it should be activated by default.
